CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## 4.1.1
+## 4.1.2
+- Remove ES 1.x from test matrix. The testing requirements were too severe, and update bugs were too troublesome to correct. See more about the update bugs @ https://github.com/elastic/elasticsearch/issues/19103
```

Comment:
Alternately, if this update really does abandon ES 1.x, but this plugin v4.1.1 *did* support ES 1.x, we should do a major version bump to 5.0.0 of this plugin to indicate that a compatibility to prior major was lost.